presto-main/src/main/java/com/facebook/presto/sql/planner/plan/Assignments.java

```diff
@@ -69,17 +71,22 @@ public static Assignments of(VariableReferenceExpression variable1, RowExpressio
         return builder().put(variable1, expression1).put(variable2, expression2).build();
     }
 
+    private final List<VariableReferenceExpression> outputVariables;
```

Comment:
Why enforce order?

Reply:
I have the same question. At least it's not obvious from the enforcing order commit why this is useful. The order is not guaranteed in anyways because all calls to `new Assignments` are replaced with `Assignments.copyOf` which just collects `outputVariables` with arbitrary order anyways.

Reply:
You still didn't answer what relies on the order being fixed? What would happen if it's not?